reporting_app/static/datatable.js

```diff
@@ -92,6 +92,18 @@ var merge_multi_sources_keep_first = function(dt_config){
     return _merge_multi_sources(dt_config, merge_on_keep_first);
 }
 
+var test_exist = function(variable){
+    if ( variable instanceof Array ) {
+        return variable.length > 0;
+    }
+    return variable !== undefined && variable !== null && variable;
```

Comment:
`variable` evaluate to `true` for non empty strings which is what I want.
0 should evaluate to `true` since `trim_2` could potentially be 0
I did not add support for object yet but could do in a latter PR.